src/detectors/ZDC/ZDC.cc

```diff
@@ -33,7 +33,7 @@ extern "C" {
             .capADC = 32768,
             .dyRangeADC = 2000 * dd4hep::MeV,
             .pedMeanADC = 400,
-            .pedSigmaADC = 3.2,
+            .pedSigmaADC = 0.2,
```

Comment:
Basically, here you claim that 15-bit ADC can not resolve your pedestal. That is unrealistic.

Reply:
The current parameters cause issues for O(5-10 MeV) photons. 5-10 MeV photons may be a reasonable lower limit target for the LYSO.

We do not have expected noise estimates yet. But the current version has a threshold cut at 4*3.2 = 12.8 ADC channels, which is a deposited energy per tower of about 0.8 MeV. For LYSO, a 0.8 MeV deposit corresponds to a light yield of about 25,000 photons.

If we don't want to decrease to pedestal width from 3.2 ADC channels, we maybe can decrease the dynamic range of the LYSO from the current 2 GeV value.

Reply:
Sounds good. Thanks. We'll chat with the DSC about the parameter values.